fitbenchmarking/fitbenchmarking
# Top level index page

#### Description of Work

Fixes #383 

Adds top level index for when multiple data sets are benchmarked

#### Testing Instructions

1. Check links work and the style of the html is consistent. 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
